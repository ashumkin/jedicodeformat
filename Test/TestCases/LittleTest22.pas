unit LittleTest22;

interface

implementation

procedure False;
begin
end;

function Null: integer;
begin
  result := 3;
end;                       

end.
 