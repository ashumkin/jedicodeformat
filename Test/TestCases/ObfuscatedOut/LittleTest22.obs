unit littletest22;interface implementation procedure false;begin end;function null
:integer;begin result:=3;end;end.