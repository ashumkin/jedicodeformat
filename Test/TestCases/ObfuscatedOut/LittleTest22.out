unit littletest22;

interface

implementation

procedure False;
begin
end;

function null: integer;
begin
  Result := 3;
end;

end.
